README.md

```diff
@@ -98,14 +103,92 @@ We have a small collection of algorithms, one for every day of the month. Scroll
       left => right
     ```
 
+###  **December 6 - LCM**
```

Comment:
Add a resources section with a link to what gcd is.
Use this one maybe: https://www.mathsisfun.com/greatest-common-factor.html